modules/core/src/v2/pendingApproval.ts

```diff
@@ -303,6 +318,37 @@ export class PendingApproval {
       .asCallback(callback);
   }
 
+  getCryptographicApprovalPayload(pendingApproval: PendingApprovalData, callback?: NodeCallback<any>): Bluebird<any> {
+    const self = this;
+    return co(function*() {
+      // this function is currently overkill for a getter, but we should in
+      // the future add validation that the payload matches the policy body.
+      // This is an async function so if we need an async validation it does
+      // not require a function signature change.
+      return pendingApproval.signingPolicy.payload;
+    }).call(this)
+      .asCallback(callback);
+  }
+
+  cryptographicApprove(params: CryptographicApproveOptions = {}, callback?: NodeCallback<any>): Bluebird<any> {
+    const self = this;
+    return co(function*() {
+      validateParams(params, [], ['walletPassphrase', 'otp'], callback);
+
+      const approvalParams: any = { 
+        state: 'approved',
+        signatures: [params.signature]
+        otp: params.otp,
+      };
+      return self.bitgo
+        .put(self.url())
+        .send(approvalParams)
+        .result()
+        .nodeify(callback);
```

Comment:
Remove the `nodeify` call here, I think this would result in the callback being called twice (since it's also passed to `asCallback` before).